<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogActions</class>
 <widget class="QDialog" name="DialogActions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Toolbar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Available Actions:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Current Actions:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButtonUp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Up</string>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
        <property name="arrowType">
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="toolButtonRemove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Delete</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButtonAdd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Add</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="toolButtonDown">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Down</string>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QListWidget" name="treeWidgetDest">
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="treeWidgetOrig">
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogActions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogActions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
